js/form_editor_controller.js

```diff
@@ -745,11 +751,16 @@ class GlpiFormEditorController
      * @returns {jQuery} Copy of the template
      */
     #copy_template(target, destination, action = "append") {
+        /* global setupAjaxDropdown */
+
```

Comment:
Accessible global variables are documented at the start of the file.
Lets keep things consistent for now (but it is a good suggestion to define them in each methods instead, we can look into it later).